core/whistle_media-1.0.0/src/whistle_media_maintenance.erl

```diff
@@ -303,36 +303,33 @@ remove_empty_media_docs(AccountId, AccountDb) ->
         {'ok', MediaDocs} ->
             io:format("found ~b media docs in account ~s~n", [length(MediaDocs), AccountId]),
             Filename = list_to_binary(["/tmp/empty_media_", AccountId, "_", wh_util:to_binary(wh_util:current_tstamp()), ".json"]),
-            {'ok', File} = file:open(Filename, ['write', 'binary']),
             io:format("archiving removed media docs to ~s~n", [Filename]),
-            remove_empty_media_docs(AccountId, AccountDb, File, MediaDocs);
+            remove_empty_media_docs(AccountId, AccountDb, Filename, MediaDocs);
         {'error', _E} ->
             io:format("error looking up media docs in account ~s: ~p~n", [AccountId, _E])
     end.
 
--spec remove_empty_media_docs(ne_binary(), ne_binary(), file:io_device(), wh_json:objects()) -> 'ok'.
-remove_empty_media_docs(AccountId, _AccountDb, File, []) ->
-    file:close(File),
+-spec remove_empty_media_docs(ne_binary(), ne_binary(), file:name(), wh_json:objects()) -> 'ok'.
+remove_empty_media_docs(AccountId, _AccountDb, _Filename, []) ->
     io:format("finished cleaning up empty media docs for account ~s~n", [AccountId]);
-remove_empty_media_docs(AccountId, AccountDb, File, [Media|MediaDocs]) ->
-    maybe_remove_media_doc(AccountDb, File, wh_json:get_value(<<"doc">>, Media)),
-    remove_empty_media_docs(AccountId, AccountDb, File, MediaDocs).
+remove_empty_media_docs(AccountId, AccountDb, Filename, [Media|MediaDocs]) ->
+    maybe_remove_media_doc(AccountDb, Filename, wh_json:get_value(<<"doc">>, Media)),
+    remove_empty_media_docs(AccountId, AccountDb, Filename, MediaDocs).
 
--spec maybe_remove_media_doc(ne_binary(), file:io_device(), wh_json:object()) -> 'ok'.
-maybe_remove_media_doc(AccountDb, File, MediaJObj) ->
+-spec maybe_remove_media_doc(ne_binary(), file:name(), wh_json:object()) -> 'ok'.
+maybe_remove_media_doc(AccountDb, Filename, MediaJObj) ->
     case wh_doc:attachments(MediaJObj) of
         'undefined' ->
             io:format("media doc ~s has no attachments, archiving and removing~n"
-                      ,[wh_json:get_value(<<"_id">>, MediaJObj)]
+                      ,[wh_doc:id(MediaJObj)]
                      ),
-            file:write(File, wh_json:encode(MediaJObj)),
-            file:write(File, [$\n]),
+            wh_util:write_file(Filename, [wh_json:encode(MediaJObj), $\n], ['append']),
```

Comment:
Why this change? Why `write_file/3` vs passing the io_device() and writing as we go?